File: Python/Basic/work1.py
'''
課題work1
a=25, b=12として、aを2乗したものにbを5回かけた値を出力するプログラムを作成しよう。

この問題はいろいろな書き方があります。下の結果はすべて同じです。
'''

a=25
b=12

c=a*a+b+b+b+b+b  #aを2回かけて、bを5回足す
print(c)
d=a*a+5*b        #aを2回かけて、bに5をかける。
print(d)
print(a**2+5*b)  #aの2乗を行い、bに5をかける。その式を直接print()関数に入れてやっても、
                 #しっかりと表示してくれる。'x**n'でxのn上を意味する。

File: C/Basic/work1.cpp
/*
課題work1
program1では変数をint型で定義しましが、work1ではdouble型にしてみましょう。
なお、計算は足し算、引き算、掛け算、割り算までで大丈夫です。余りの計算はdouble型ではエラーが出てしまいます。
数値はa=5,b=3です。
ヒント：printf内の変換文字をintでは%dとしましたが、doubleでは%fにしてみましょう。
*/
#include<stdio.h>
int main(void){
  double a=5,b=3;
  printf("a+b=%f\n",a+b);//aとbの足し算
  printf("a-b=%f\n",a-b);//aとbの引き算
  printf("a×b=%f\n",a*b);//aとbの掛け算
  printf("a÷b=%f\n",a/b);//aとbの割り算
  return 0;
}
